### PR TITLE
Update benchmarks

### PR DESCRIPTION
**Pull request by boopathi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,31 +179,30 @@ Add to your `.babelrc`'s plugins array.
...
 | --------------------- | ----------- | ------- | ----------- | -------- | --------------- | ---------------- |
-| **babili**            | 93.49kB     | 62%     | 34.81kB     | 49%      | 7.18            | 6370.95          |
-| **uglify**            | 93.39kB     | 62%     | 34.81kB     | 49%      | 7.30            | 1987.65          |
-| **closureCompiler**   | 91.59kB     | 63%     | 34.65kB     | 49%      | 8.88            | 5473.73          |
-| **closureCompilerJs** | 94.76kB     | 62%     | 35.59kB     | 48%      | 7.64            | 11745.57         |
-| **butternut**         | **88.73kB** | **64%** | **32.5kB**  | **53%**  | **8.56**        | **630.11**       |
+| **babelMinify**       | 97.23kB     | 62%     | 36.09kB     | 49%      | 6.49            | 4996.68          |
+| **uglify**            | 96.95kB     | 63%     | 35.99kB     | 49%      | 5.80            | 1478.59          |
+| **closureCompiler**   | 95.29kB     | 63%     | 35.8kB      | 50%      | 8.97            | 3111.51          |
+| **closureCompilerJs** | 98.53kB     | 62%     | 36.73kB     | 48%      | 5.87            | 9386.95          |
```
Comment (vigneshshanmugam) [inline README.md]: what happened to butternut result?

File: README.md [hunk 2]
```diff
@@ -179,31 +179,30 @@ Add to your `.babelrc`'s plugins array.
...
-| minifier              | output raw | raw win | gzip output | gzip win | parse time (ms) | minify time (ms) |
-| --------------------- | ---------- | ------- | ----------- | -------- | --------------- | ---------------- |
-| **babili**            | 37.89kB    | 72%     | 12.54kB     | 60%      | 2.39            | 2262.64          |
-| **uglify**            | **37.7kB** | **72%** | **12.4kB**  | **60%**  | **2.68**        | **1019.53**      |
-| **closureCompiler**   | 36.73kB    | 72%     | 12.45kB     | 60%      | 3.01            | 2692.87          |
-| **closureCompilerJs** | 44.58kB    | 67%     | 14.54kB     | 53%      | 2.89            | 6591.58          |
-| **butternut**         | 39.2kB     | 71%     | 12.81kB     | 59%      | 2.88            | 281.72           |
+| minifier              | output raw  | raw win | gzip output | gzip win | parse time (ms) | minify time (ms) |
+| --------------------- | ----------- | ------- | ----------- | -------- | --------------- | ---------------- |
+| **babelMinify**       | 40.63kB     | 71%     | 13.33kB     | 60%      | 2.40            | 1879.97          |
```
Comment (Andarist) [inline README.md]: shouldnt it be listed as `babel-minify`? Would look better and correspond to the actual name of the package

File: README.md (hunk 1 above)
Comment (boopathi) [inline README.md]: The result is wrong. https://github.com/Rich-Harris/butternut/issues/162

File: README.md (hunk 2 above)
Comment (boopathi) [inline README.md]: 👍

**Pull request merged**